components/Chat/index.vue

```diff
@@ -18,30 +18,30 @@
 
       <!-- Show Poll to Users -->
       <v-slide-x-reverse-transition>
-        <chat-poll-vote
-          v-if="showPollClient"
-          :poll-data="pollData"
-          :is-owner="pollData.owner === ( user ? user.uid : null )"
-          @vote="votePoll"
-          @end="endPoll"
-          @destroy="destroyPoll"
-        />
+	<chat-poll-vote
+	  v-if="showPollClient"
+	  :poll-data="pollData"
+	  :is-owner="pollData.owner === ( user ? user.uid : null )"
+	  @vote="votePoll"
+	  @end="endPoll"
+	  @destroy="destroyPoll"
+	/>
```

Comment:
Oh jeesh, this is awful. It might be the correct spacing tho, I can't really tell.